lib/build.js

```diff
@@ -8,7 +8,16 @@ const pify = require('pify');
 const writeFile = pify(fs.writeFile);
 
 module.exports = context => {
-	const dockerfile = `FROM node:${context.version}-onbuild`;
+	const dockerfile = [
+		`FROM node:${context.version}`,
+		'RUN mkdir -p /usr/src/app',
+		'WORKDIR /usr/src/app',
+		'COPY package.json /usr/src/app/',
```

Comment:
Since you're in `WORKDIR` folder, you can write `COPY package.json ./`.

Reply:
Good call, change made.